Revert add_dummy_time_to_wl code to previous method

elehmer opened the pull request:
## Summary of changes and related issue

Revert `add_dummy_time_to_wl` function to original design. Function was changed to look for `time_delta` instead of `{time_freq}_from_center`.  `WarmingLevels` `sliced_data` DataArray uses this later time dimension.

## Relevant motivation and context

Fixes broken `warming_level_approach` notebook.

## How to test 

Run the `warming_level_approach` notebook with this branch loaded for `climakitae`. Plus load the `fix_wl_plot2` branch for `climakitaegui` to fix the visualization plots.

## Type of change
- [ x ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
- [ ] None of the above  

## To-Do
- [ ] Unit tests
  - [ ] Existing unit tests are passing
  - [ ] If relevant, new unit tests are written (required 80% unit test coverage)
- [ ] Documentation
  - [ ] Intent of all functions included
  - [ x ] Complex code commented
  - [ ] Functions include [NumPy style docstrings](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_numpy.html) 
- [ ] Naming conventions followed
  - [ ] Helper functions hidden with `_` before the name
- [ x ] Any notebooks known to utilize the affected functions are still working
- [ x ] Black formatting has been utilized
- [ ] Tagged/notified 2 reviewers for this PR

elehmer commented:
I have modified the code to take an optional parameter `time_slice=True/False` that will modify the behavior of the util func

neilSchroeder commented on climakitae/util/utils.py:
```diff
@@ -868,21 +868,35 @@ def add_dummy_time_to_wl(wl_da):
+    if time_delta:
+        time_freq_name = wl_da.frequency
+    else:
+        time_freq_name = wl_time_dim.split("_")[0]
```
Is there a reason for not doing:

```
if time_delta:
    # set variables
    wl_time_dim = "time_delta"
    time_freq_name = wl_da.frequency
     name_to_freq = {"hourly": "h", "daily": "D", "monthly": "ME"}
else:
    wl_time_dim = [dim for dim in wl_da.dims if "from_center" in dim][0]
    time_freq_name = wl_time_dim.split("_")[0]
    name_to_freq = {"hours": "H", "days": "D", "months": "ME"}
```

neilSchroeder replied:
Otherwise this all looks fine.

elehmer commented:
> I have modified the code to take an optional parameter `time_slice=True/False` that will modify the behavior of the util func

I removed this extra parameter and it now senses based on the data inputted.

elehmer commented:
Ideally in the future the data structures of the warming level data will be consistent, using `time_delta`. This can be part of the warming levels notebook rework.

elehmer commented:
Going to merge this. We can improve it when we rework Warming Levels
